pages/info/[memberCode].tsx

```diff
@@ -68,9 +67,9 @@ export default MemberInfo
 export const getServerSideProps: GetServerSideProps = async context => {
   const { memberCode } = context.query
 
-  const { user, error } = await getMemberByMemberCode(memberCode as string)
+  const res = await getMemberByMemberCode(memberCode as string)
```

Comment:
Is this "errors' being handled?

Reply:
Here I show error, and a return home link
![image](https://user-images.githubusercontent.com/61360270/201200716-4b5adf11-29ce-450c-b3f2-8cc64078e771.png)